front/components/ideaDetail/IdeaShow.vue

```diff
@@ -134,7 +134,7 @@ export default {
         }
       }
 
-      return content
+      return `${content}<br><br>`
```

Comment:
thoughts on using css margin instead of <br> for creating whitespace on the bottom?

Reply:
That is what I was trying to do originally and would be ideal, but I was having issues. The content comes wrapped in a <div> that I cannot seem to access through CSS. I would have expected something like this to work in IdeaContent.vue

```
.content {
   div {
     margin-bottom: 2rem;
   }
}
```

But it does not seem to add the margin to the content. Tried the same thing with padding and it does not seem to apply that either. This is the only way I could get it to work. Not sure what I am doing wrong.

Reply:
@f-prime f it. it works. merging.

:)

I care FIRST about it working and SECOND about beauty.